templates/includes/sidebar.html

```diff
@@ -0,0 +1,8 @@
+<div class="sidebar">
+    <ul>
+        <li><a href="#">Home page</a></li>
+        <li><a href="#">Manufacturers</a></li>
+        <li><a href="#">Cars</a></li>
+        <li><a href="#">Drivers</a></li>
```

Comment:
The links in the sidebar are currently using `#` as the URL, which is a placeholder. You should replace these with actual URLs or Django URL template tags to link to the appropriate pages in your application.

Reply:
I can use it, check the README file.
Anchor tags can serve as placeholder links for this task, meaning the actual destination of the link is not a concern. For example, you can use href="#" as the link destination.